queries.sql

```diff
@@ -5,6 +5,47 @@ SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-0
 SELECT name FROM animals WHERE neutered IS true AND escape_attempts < 3;
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered IS true;
+SELECT * FROM animals WHERE neutered IS TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg >= 10.4  AND weight_kg <= 17.3;
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN WORK;
+    UPDATE animals SET species = 'unspecified';
+
+SELECT species FROM animals;
+
+ROLLBACK;
+
+SELECT species FROM animals;
+
+BEGIN WORK;
+
+UPDATE animals SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
+SELECT species FROM animals;
+
+BEGIN WORK;
+    DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+    SAVEPOINT SP1;
+    UPDATE animals SET weight_kg = weight_kg * -1;
+    
+    ROLLBACK TO SP1;
+
+    UPDATE animals SET weight_kg = weight_kg * -1
+    WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] There is a transaction before this one `Inside a transaction delete all records in the animals table, then roll back the transaction.` Kindly add it.